modules/nuxeo-apidoc-webengine/pom.xml

```diff
@@ -42,6 +42,10 @@
       <groupId>javax.transaction</groupId>
       <artifactId>javax.transaction-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>es.nitaur.markdown</groupId>
+      <artifactId>txtmark</artifactId>
```

Comment:
this seems to be quite old... perhaps something like https://github.com/atlassian/commonmark-java would be a better option

Reply:
I reviewed a few of them, and txtmark was praised for its performance. I was hesitating on the version though (original last version is 0.13, this one is a fork, with version 0.16).
Also i don't think markdown has been revolutionized lately so i guess it's not that important to gave something very recent.
About commonmark, i'm not sure about the license https://github.com/atlassian/commonmark-java/blob/master/LICENSE.txt

Reply:
BSD 2-Clause is ok, we already have a number of them, see https://doc.nuxeo.com/nxdoc/licenses/

Reply:
ok then i'll have a try at this other lib, see how it goes :+1:

Reply:
so i have a strange behaviour on lists with this one (paragraphs imbricated and such), i do have some problems with the first library still, but this one seems worse... github readme rendering is much better in both cases.
i'm testing with the following, maybe i should check for correct markdown before testing again with both converters?

- https://github.com/nuxeo/nuxeo/blob/master/modules/platform/nuxeo-platform-lang-ext/README.md
- https://github.com/nuxeo/nuxeo-explorer/blob/master/ReadMe.md
- https://github.com/nuxeo/nuxeo-explorer/blob/master/modules/nuxeo-apidoc-core/ReadMe.md

I'll add the "preview" label to the PR in any case, to check final rendering.

Reply:
see #32 (labelled "preview" too) for commonmark rendering instead

Reply:
To be compared:
txtmark:
- https://preview-nuxeo-explorer-pr-31.platform.dev.nuxeo.com/nuxeo/site/distribution/server-11.3.15/viewBundle/org.nuxeo.apidoc.core
- https://preview-nuxeo-explorer-pr-31.platform.dev.nuxeo.com/nuxeo/site/distribution/server-11.3.15/viewBundle/org.nuxeo.ecm.platform.lang.ext

vs commonmark:
- https://preview-nuxeo-explorer-pr-32.platform.dev.nuxeo.com/nuxeo/site/distribution/server-11.3.15/viewBundle/org.nuxeo.apidoc.core
- https://preview-nuxeo-explorer-pr-32.platform.dev.nuxeo.com/nuxeo/site/distribution/server-11.3.15/viewBundle/org.nuxeo.ecm.platform.lang.ext

In both cases github rendering of markdown is much better, but if these are considered invalid, i can also modify the apidoc modules readmes for a better rendering in this case.

Reply:
After checking more closely, commonmark rendering is indeed better.
I will adjust the CSS at render because it interferes with the final display, which could be nicer.

Reply:
Resolving as rendering is much better now, i just need to handle some GH markdown use cases before i can push new changes.